docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -79,55 +103,35 @@ i.e. you cannot bind a Horizontal Pod Autoscaler to a replication controller and
 The reason this doesn't work is that when rolling update creates a new replication controller,
 the Horizontal Pod Autoscaler will not be bound to the new replication controller.
 
-## Support for custom metrics
-
-Kubernetes 1.2 adds alpha support for scaling based on application-specific metrics like QPS (queries per second) or average request latency.
-
-### Prerequisites
+## Support for multiple metrics
 
-The cluster has to be started with `ENABLE_CUSTOM_METRICS` environment variable set to `true`.
+Kubernetes 1.6 adds support for scaling based on multiple metrics. You can use the `autoscaling/v2alpha1` API
```

Comment:
no, support for multiple metrics is turned on by default, effectively, as long as the autoscaling/v2alpha1 API is enabled (or you use the equivalent annotations in v1).